src/main/java/com/faforever/client/preferences/PreferencesService.java

```diff
@@ -346,6 +355,22 @@ public Path getThemesDirectory() {
     return getFafDataDirectory().resolve("themes");
   }
 
+  public Path getNewGameLogFile() {
+    try (Stream<Path> listOfLogFiles = Files.list(getFafLogDirectory())) {
+      listOfLogFiles
+          .filter(p -> GAME_LOG_PATTERN.matcher(p.getFileName().toString()).matches())
+          .sorted(Comparator.comparingLong(p -> p.toFile().lastModified()))
+          .sorted(Collections.reverseOrder())
+          .skip(NUMBER_GAME_LOGS_STORED - 1)
+          .forEach(p -> noCatch(() -> Files.delete(p)));
+    } catch (IOException e) {
+      logger.error("Could not list log directory.", e);
+    } catch (NoCatchException e) {
+      logger.error("Could not delete game log file");
+    }
+    return getFafLogDirectory().resolve(new SimpleDateFormat("'game_'yyyy-MM-dd_HHmmss'.log'").format(new Date()));
```

Comment:
How is that gonna look are there no seperation between Hours and minutes?

Reply:
![Screenshot (50)](https://user-images.githubusercontent.com/66929319/91646692-e714b080-ea1f-11ea-896f-8bcded2ff2f1.png)

Thought this was a standard time format but can be changed if you have one you like better that is file valid

Reply:
idk I find it hard to read. Btw the old client used the game uuid as a name. Not sure of that would be an advantage. I think that was so you could match replay and game log.

Reply:
That might be better I will just change to that if it works for you

Reply:
Yeah sure. This is not what we gotta do, I also like the time. I am indecisive.

Reply:
Yeah personally I think the time might be more intuitive for the user than game uid and I can add a dash separator for the time